.github/workflows/main.yml

```diff
@@ -93,7 +93,7 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm run clean
-      - run: cd server && npm install
+      - run: npm install && npm run fetch:docs
```

Comment:
It used to be called by npm pack the line bellow.
I think the change is good, but could you just explain it to me briefly?